test/lib/ufe/testSelection.py

```diff
@@ -445,5 +445,53 @@ def testMayaSelectSwitchVariant(self):
 
         self.assertTrue(sn.empty())
 
+    @unittest.skipUnless(mayaUtils.previewReleaseVersion() >= 125, 'Requires Maya fixes only available in Maya Preview Release 125 or later.')
+    def testMayaSelectUndoPrimCreation(self):
+        '''Test if the SceneItem's prim is still valid on selection after the prim creation is undo than redo'''
+
+        # helper function to check if the current
+        # selected SceneItem's prim is valid.
+        def checkSelectedSceneItemPrim():
```

Comment:
Why not pass in the scene item that you expect will be the selected item, then compare it with globalSn.front()?  Would make for an even stronger test.